### ISSUE TITLE
rule ban:  Hot to set an option for an array?

### ISSUE THREAD

**Issue by AnteaterKit** [palantir/tslint#3885]
### Question

- __TSLint version__: 5.10.0
- __TypeScript version__: 2.7.2
- __Running TSLint via__: (pick one) VSCode

#### TypeScript code being linted

```ts
function getDefaultState () {
    return {
        isLoading: false,
        page: CoreUtils.find('p1', 1)
    };
}

function get(relatives: Relative[]){
    const relative = relatives ? relatives.find(element => element.guid === Id);
}
```

with `tslint.json` configuration:

```
 "ban": [
            true,
            "eval",
            {"type": "array", "name": ["*", "find"], "message": "Use a regular for  instead."}
        ]
```

#### Actual behavior
getDefaultState: Calls to '*.find' are not allowed. Use a regular for  instead.
get:  Calls to '*.find' are not allowed. Use a regular for  instead.
#### Expected behavior
get:  Calls to '*.find' are not allowed. Use a regular for  instead.

Hot to set an option for an array?

**Comment by giladgray** [comment]
@AnteaterKit just pass an `[array]`: https://github.com/palantir/blueprint/blob/develop/packages/tslint-config/index.js#L18-L29

```json
        "ban": {
            "options": [
                ["_", "extend", "use object spread: { ...a, ...b }"],
                ["_", "isNull", "use plain JS: == null"],
```

**Comment by giladgray** [comment]
the docs are full of usage examples, especially for [this `ban` rule](https://palantir.github.io/tslint/rules/ban/)

**Comment by AnteaterKit** [comment]
@giladgray  thank you!
